lib/rubocop/ast/node/retry_node.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module AST
+    # A node extension for `retry` nodes. This will be used in place of a
+    # plain node when the builder constructs the AST, making its methods
+    # available to all `retry` nodes within RuboCop.
+    class RetryNode < Node
+      include MethodDispatchNode
+    end
```

Comment:
This looks like a wonderful approach 🌟 

However issue #6751 also reports on `raise` and `break`.  `raise` seems to work without errors, but `break` seems to be an error. 

## `raise` (no offenses)

```console
% cat raise.rb
if true then raise else 7 end
% rubocop -a --only Style/OneLineConditional raise.rb
Inspecting 1 file
C

Offenses:

raise.rb:1:1: C: [Corrected] Style/OneLineConditional: Favor the ternary operator (?:) over if/then/else/end constructs.
if true then raise else 7 end
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

1 file inspected, 1 offense detected, 1 offense corrected
% g diff raise.rb
diff --git a/raise.rb b/raise.rb
index 519c3b0..69ac23b 100644
--- a/raise.rb
+++ b/raise.rb
@@ -1 +1 @@
-if true then raise else 7 end
+true ? raise : 7
```

## `break` (offense detected)

```console
% cat break.rb
if true then break else 7 end
% rubocop -a --only Style/OneLineConditional break.rb
Inspecting 1 file


0 files inspected, no offenses detected
undefined method `prefix_not?' for s(:break):RuboCop::AST::Node
/Users/koic/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/rubocop-0.64.0/lib/rubocop/cop/style/one_line_conditional.rb:92:in `keyword_with_cha
nged_precedence?'
/Users/koic/.rbenv/versions/2.6.1/lib/ruby/gems/2.6.0/gems/rubocop-0.64.0/lib/rubocop/cop/style/one_line_conditional.rb:80:in `requires_parenth
eses?'
```

So I think that it would be better to also make `BreakNode`. WDYT?

Reply:
> So I think that it would be better to also make `BreakNode`. WDYT?

👍